README.md

```diff
@@ -1,76 +1,25 @@
 # Hardware classification controller
 
```

Comment:
I would like to mention the goal of HWCC, what you mentioned below is more of implementation details. First elaborate what's goal of the project then you can keep below "how" part as is.